Appraisals

```diff
@@ -5,3 +5,7 @@ end
 appraise 'rails-6-1' do
   gem 'railties', '~> 6.1.0'
 end
+
+appraise 'rails-7.0' do
```

Comment:
```suggestion
appraise 'rails-7-0' do
```

Reply:
Whoops! Seems to do the same thing, but I'll fix this for consistency!

Reply:
It's generating slightly different filenames, yeah -- `7.0.gemfile` vs `7_0.gemfile`